### PR TITLE
[WIP] Volunteering Page

### PR DESCRIPTION
**Pull request by BNewing**
Draft of a new page describing how to volunteer with Zooniverse and what the different volunteering roles are. Will fit in the About section as a new section.

Will go over the draft with @mrniaboc next week to go over the info in it

### CODE REVIEW

**Comment by rogerhutchings** [comment]
Here's a similar issue, if you're feeling keen - #2526

**Comment by BNewing** [comment]
I am indeed feeling keen :D

File: app/pages/about/volunteering-page.cjsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  volunteeringPage:
+   pageContent: '''
+    ## How to Volunteer
+
+    There are several different ways to volunteer with the Zooniverse. The main ways are helping us with classifications on data, being a beta tester on projects we've yet to launch, and being a moderator for a project. For more information about any of these roles, just read below.
+
+    ### Volunteer on a Project
+    This is the easiest and most common way of volunteering with the Zooniverse! We always need volunteers to go onto our projects and classify the data contained in them. You can read more about what happens with the classifications and how it helps the scientific community and the progress of science on the [About](/about) page.
+
+    There is no minimum time requirement needed; do as much or as a little as you'd like. To get started as a classifications volunteer, just to go to the [Projects](/projects) page, have a look through, find one you like the look of, and get stuck in!
```
Comment (alexbfree) [inline app/pages/about/volunteering-page.cjsx]: there's a typo here: should be "or as little"

File: app/pages/about/volunteering-page.cjsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    ### Volunteer as a Beta Tester
+    Volunteers also help us test projects before they are launched to check that they work properly. This involves running through the beta project, looking for any bugs, and filling out a questionnaire at the end. This helps us find any issues in the project that need resolving and also assess how suitable the project is for the Zooniverse.
```
Comment (alexbfree) [inline app/pages/about/volunteering-page.cjsx]: "running through" is a bit vague.

File: app/pages/about/volunteering-page.cjsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+    To sign up as a beta tester, just go to your profile page, select the e-mail tab and tick the box next relating to beta testing. We'll then send you emails when a project is ready to be tested. If you'd like to stop being a beta tester, and stop receiving these emails, just go back to the same page and untick the box.
+
+    ### Volunteer as a Project Moderator
+    Volunteer moderators have extra permissions in the talk board for a particular project. They help moderate discussions and act as a point of contact for the project. Volunteer moderators are selected by the project owner.
```
Comment (alexbfree) [inline app/pages/about/volunteering-page.cjsx]: Talk should have a capital T

Comment (alexbfree) [inline app/pages/about/volunteering-page.cjsx]: Need to say how people volunteer in this way ... how to find the project owner and how to message them?

**Comment by rogerhutchings** [comment]
Superseded by #3040 3040

**Comment by BNewing** [comment]
I thought I'd rename the branch as I'm working on another project that actually is a glossary page, and this wasn't named so well. I tested out what I thought was a method for simply renaming a project and tested it on my to-do list twice - but didn't check it against anything with a PR against it, and then in the process learnt that it deletes the branch to do the rename - hence the new PR. Sorry! 

Thank you for the suggestions @alexbfree  I'll add them in now!

**Pull request closed without merging**